stream_test.go

```diff
@@ -487,3 +488,89 @@ func TestStreamWriterGetRowElement(t *testing.T) {
 		assert.False(t, ok)
 	}
 }
+
+func TestSpecificTempDirStreamingWriter(t *testing.T) {
+	checkTempFileExist := func(testName, tempDir string, fileNames []string, exist bool) {
```

Comment:
The `filenames` are always empty, this test case can be removed, update the `TestStreamWriter` function to cover the custom temporary directory.